ui/chart/src/common.ts

```diff
@@ -9,11 +9,13 @@ export interface MovePoint {
 
 let highchartsPromise: Promise<any> | undefined;
 
-export function selectPly(this: PlyChart, ply: number | false) {
-  if (this.lastPly === ply) return;
+export function selectPly(this: PlyChart, ply: number, onMainline: boolean) {
+  if (this.lastPly === ply && (this as any).onMainline == onMainline) return;
```

Comment:
`as any` :thinking: do we need this?

Reply:
No we do not.  I would just remove the entire check, the short circuit doesn't save us enough to be worth the complexity.